src/mmw/js/src/modeling/templates/scenarioToolbarTabContent.html

```diff
@@ -15,18 +15,24 @@
             <div id="modifications" class="dropdown-menu menu-right" role="menu"> <!-- DropdownContent -->
                 {% for controlName, models in groupedShapes %}
                     <div class="pad-1 brd-bottom">
-                        <label class="medium">
-                            {% if controlName == 'landcover' %}
-                                Land Cover
-                            {% elif controlName == 'conservation_practice' %}
-                                Conservation Practice
-                            {% endif %}
-                        </label>
                         <table id="mod-{{ controlName|lower|replace(' ', '') }}" class="table modifications custom-hover">
+                            <thead>
+                                <tr>
+                                    <td class="text-left modification-type">
+                                        {% if controlName == 'landcover' %}
+                                            Land Cover
+                                        {% elif controlName == 'conservation_practice' %}
+                                            Conservation Practice
+                                        {% endif %}
+                                    </td>
+                                    <td class="text-right">Effective Area</td>
+                                    <td></td>
+                                </tr>
+                            </thead>
                             {% for model in models %}
                                 <tr>
                                     <td>{{ model.get('value')|tr55Name }}</td>
-                                    <td class="strong text-right">{{ model.get('area')|round(1)|toLocaleString }} {{ model.get('units') }}</td>
+                                    <td class="strong text-right">{{ model.get('effectiveArea')|round(1)|toLocaleString }} {{ model.get('effectiveUnits') }}</td>
```

Comment:
Might want to use `round(2)|toLocaleString(2)` here too.

I wonder if we can forego `round` and just use `toLocaleString`? But that may clip instead of round.